content/id/docs/concepts/overview/object-management-kubectl/declarative-config.md

```diff
@@ -0,0 +1,868 @@
+      "spec":{"containers":[{"image":"nginx:1.7.9","name":"nginx",
+      "ports":[{"containerPort":80}]}]}}}}
+  # ...
+spec:
+  # ...
+  minReadySeconds: 5
+  selector:
+    matchLabels:
+      # ...
+      app: nginx
+  template:
+    metadata:
+      # ...
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - image: nginx:1.7.9
+        # ...
+        name: nginx
+        ports:
+        - containerPort: 80
+        # ...
+      # ...
+    # ...
+  # ...
+```
+
+Perbarui nilai `replicas` secara langsung di konfigurasi `live` dengan menggunakan perintah `kubectl scale`. Pembaruan ini tidak menggunakan perintah `kubectl apply`:
+
+```shell
+kubectl scale deployment/nginx-deployment --replicas=2
+```
+
+Tampilkan konfigurasi *live* dengan perintah `kubectl get`:
+
+```shell
+kubectl get -f https://k8s.io/examples/application/simple_deployment.yaml -o yaml
+```
+
+Keluaran perintah di atas akan menunjukkan bahwa nilai `replicas` telah diubah menjadi 2, dan anotasi `last-applied-configuration` tidak memuat nilai `replicas`:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    # ...
+    # perhatikan bahwa anotasi tidak memuat nilai replicas
+    # karena nilai tersebut tidak diperbarui melalui perintah kubectl-apply
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"apps/v1","kind":"Deployment",
+      "metadata":{"annotations":{},"name":"nginx-deployment","namespace":"default"},
+      "spec":{"minReadySeconds":5,"selector":{"matchLabels":{"app":nginx}},"template":{"metadata":{"labels":{"app":"nginx"}},
+      "spec":{"containers":[{"image":"nginx:1.7.9","name":"nginx",
+      "ports":[{"containerPort":80}]}]}}}}
+  # ...
+spec:
+  replicas: 2 # ditulis oleh perintah kubectl scale
+  # ...
+  minReadySeconds: 5
+  selector:
+    matchLabels:
+      # ...
+      app: nginx
+  template:
+    metadata:
+      # ...
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - image: nginx:1.7.9
+        # ...
+        name: nginx
+        ports:
+        - containerPort: 80
+      # ...
+```
+
+Perbarui *file* konfigurasi `simple_deployment.yaml`, ubah *image* dari `nginx:1.7.9` ke `nginx:1.11.9`, dan hapus *field* `minReadySeconds`:
+
+{{< codenew file="application/update_deployment.yaml" >}}
+
+Terapkan perubahan yang telah dibuat di *file* konfigurasi:
+
+```shell
+kubectl diff -f https://k8s.io/examples/application/update_deployment.yaml
+kubectl apply -f https://k8s.io/examples/application/update_deployment.yaml
+```
+
+Tampilkan konfigurasi *live* dengan perintah `kubectl get`:
+
+```
+kubectl get -f https://k8s.io/examples/application/simple_deployment.yaml -o yaml
+```
+
+Keluaran perintah di atas akan menunjukkan perubahan-perubahan berikut pada konfiguasi *live*:
+
+- *Field* `replicas` tetap bernilai 2 sesuai dengan nilai yang diatur oleh perintah `kubectl scale`. Hal ini karena *field* `replicas` dihapuskan dari *file* konfigurasi.
```

Comment:
Will we use replicas or replika for subsequent changes?

Reply:
Because it's the actual name of the field in yaml file, I think it shouldn't be translated in this case. But, ya, for generic appearance of "replica" we can use "replika".

Reply:
then can you please put this in the glossary? 🙇

Reply:
Added.